src/Layouts/NO_QWERTY.ts

```diff
@@ -0,0 +1,43 @@
+import {LayoutMap} from './Layout';
+
+export const id = 'NO QWERTY';
```

Comment:
I'm not familiar with ISO3166 code ;)
This is great. I think a comment above the `export const id` line would be enough clear for contributors.
```
# ISO3166 country code: https://www.iso.org/obp/ui/#search/code/
export const id = 'NO QWERTY';
```